examples/js/nextjs-new/lib/bugsnag.js

```diff
@@ -0,0 +1,21 @@
+import Bugsnag from '@bugsnag/js'
+import BugsnagPluginAwsLambda from '@bugsnag/plugin-aws-lambda'
+
+export function start() {
+  // TODO NEXT_PHASE seems to be undocumented. We're doing this because we don't want
+  // Bugsnag to run during the build phase
```

Comment:
Looks like phases are slightly more documented now. See https://nextjs.org/docs/api-reference/next.config.js/introduction and https://github.com/vercel/next.js/blob/canary/packages/next/shared/lib/constants.ts#L1-L5